pdc-client.spec

```diff
@@ -142,11 +142,8 @@ find -name '*.py' | xargs sed -i '1s|^#!python|#!%{__python}|'
 export PDC_CLIENT_PLUGIN_PATH="%{buildroot}%{plugin_install_path}"
 test -d "$PDC_CLIENT_PLUGIN_PATH"
 
-%{__python2} setup.py test -q
-
 %if 0%{?with_python3}
```

Comment:
I think we can remove the following several lines

Reply:
cool! that's correct, removed, thx @simozhan